.github/workflows/contrib-tests.yml

```diff
@@ -45,6 +46,20 @@ jobs:
       - name: Test RetrieveChat
```

Comment:
Maybe add a condition to avoid run the same tests twice.

`if: matrix.python-version != '3.10'`